packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -281,7 +281,8 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
                 //for the purpose of the test, we are settings limit to 2, and fetching all.
                 //setting mirror node limit to 2 for this test only
                 process.env['MIRROR_NODE_LIMIT_PARAM'] = '2';
-                const blocksBehindLatest = Number(await relay.call('eth_blockNumber', [], requestId)) - 40;
+                // 10 is an arbitrary number and might be changed in the future for a smaller number in case test fails due to negative number in the fromBlock param
+                const blocksBehindLatest = Number(await relay.call('eth_blockNumber', [], requestId)) - 10;
```

Comment:
Is it worth making sure the block count is always above this arbitrary number to avoid a negative number as you noted?
```suggestion
                const blockOffset = 10;
                const latestBlockNum = await relay.call('eth_blockNumber', [], requestId))
                const minBlockNum = min(latestBlockNum, blockOffset); // not sure if a further offset of 1 is needed
                const blocksBehindLatest = minBlockNum - blockOffset;
```

Reply:
I've changed it to check if CurrentBlock is bigger than 10, otherwise fetch from block 0 (the beginning).